recipes-wpe/wpeframework/wpeframework_git.bb

```diff
@@ -115,6 +117,7 @@ do_install_append() {
 
     if ${@bb.utils.contains("PACKAGECONFIG", "opencdm", "true", "false", d)}
     then
+        #install -d ${STAGING_INCDIR}
```

Comment:
You'll need this to move the header if the directoy is not created earlier in the install step. This might break rebuilds if you do.

Reply:
I think this came in with a cherry-pick and the master has it as well... 

https://github.com/WebPlatformForEmbedded/meta-wpe/blob/master/recipes-wpe/wpeframework/wpeframework_git.bb#L120

Reply:
it also removed with this commit:
https://github.com/WebPlatformForEmbedded/meta-wpe/commit/43cac01fbe7568f6f5fc2bf272a5d731641b2dc4